aircraft/C130/C130.xml

```diff
@@ -112,7 +112,13 @@
     </ground_reactions>
     <propulsion>
         <engine file="t56">
+            <location unit="IN">
+                <x> 586.5 </x>
+                <y> -265 </y>
+                <z> -40 </z>
+            </location>
```

Comment:
This code is useless and has been removed by the PR #562  (commit 150805eae056b37a169d75aa6971133f01b9747f).
Please remove that change from your PR.